lib/buildr_plus/features/less.rb

```diff
@@ -17,17 +17,21 @@
     def default_less_path
       'src/main/webapp/less'
     end
+
+    attr_accessor :options
+
+    def less_path
```

Comment:
I am not sure there is value in this. Rather than being global to all projects should this not be per-project?